bin/setup

```diff
@@ -18,6 +18,7 @@ chdir APP_ROOT do
   system! 'gem install bundler --conservative'
   system('bundle check') || system!('bundle install')
 
+
```

Comment:
Is this extra blank line intentional?

Reply:
These blank lines were added by the Rails 5 task upgrading to the new binstubs. I tried removing them and running it again, they were added back. 😐